scripts/zf_mkdist

```diff
@@ -98,7 +98,12 @@ build_and_copy_artifacts() {
     fi
 
     build_dir="${zf_tmpdir}"/build
-    rm -rf "${build_dir}"
+    find "${build_dir}" -maxdepth 1 -mindepth 1 -type d |
```

Comment:
Shall we clarify in a comment why we want to remove all directories except `artifacts`? Perhaps, I'm late for the ZF party, and things are obvious to everyone else!